argo-cd-apps/base/member/infra-deployments/kustomization.yaml

```diff
@@ -21,5 +21,6 @@ resources:
   - tracing-workload-otel-collector
   - tempo
   - notification-controller
+  - kubearchive
```

Comment:
adding this to member is what we want but this member folder gets included in dev, staging and prod overlays. Since this change is about dev and staging, you need to delete the application from prod internal and external overlays, since we are not ready to get this deployed in prod. Here is the diff of how to do that:

```
git diff
diff --git a/argo-cd-apps/overlays/konflux-public-production/delete-applications.yaml b/argo-cd-apps/overlays/konflux-public-production/delete-applications.yaml
index 588724c3..4d9b0785 100644
--- a/argo-cd-apps/overlays/konflux-public-production/delete-applications.yaml
+++ b/argo-cd-apps/overlays/konflux-public-production/delete-applications.yaml
@@ -11,3 +11,9 @@ kind: ApplicationSet
 metadata:
   name: tempo
 $patch: delete
+---
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: kubearchive
+$patch: delete
diff --git a/argo-cd-apps/overlays/production-downstream/delete-applications.yaml b/argo-cd-apps/overlays/production-downstream/delete-applications.yaml
index 139b6bb0..31a58229 100644
--- a/argo-cd-apps/overlays/production-downstream/delete-applications.yaml
+++ b/argo-cd-apps/overlays/production-downstream/delete-applications.yaml
@@ -29,3 +29,9 @@ kind: ApplicationSet
 metadata:
   name: quality-dashboard
 $patch: delete
+---
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: kubearchive
+$patch: delete

```

Reply:
Good catch. Added the delete to both places with a comment.